thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datasource/pinot/PinotDataSourceDimensionFilters.java

```diff
@@ -28,7 +31,9 @@
  */
 public class PinotDataSourceDimensionFilters {
   private static final Logger LOG = LoggerFactory.getLogger(PinotDataSourceDimensionFilters.class);
-  private static final ExecutorService executorService = Executors.newFixedThreadPool(10);
+  private static final ExecutorService executorService = Executors.newFixedThreadPool(40);
```

Comment:
maybe use a cached thread pool at this point. 40 idle threads is a lot.

Reply:
Nice suggestion! Changed!